doc/classes/BaseButton.xml

```diff
@@ -39,6 +39,15 @@
 				Returns [code]true[/code] if the mouse has entered the button and has not left it yet.
 			</description>
 		</method>
+		<method name="set_pressed_visual">
+			<return type="void">
+			</return>
+			<argument index="0" name="pressed" type="bool">
+			</argument>
+			<description>
+				Changes the [member pressed] state of the button, without emitting [signal toggled]. Use only when you want to change the visual state of the button (e.g. when initializing scene). Only works if [member toggle_mode] is [code]true[/code].
```

Comment:
I'm a bit confused with what this does, and the `set_pressed_visual` name. The name is not extremely clear but it sounds to me that it would just change the visuals (i.e. use the pressed stylebox instead of the normal one, or vice versa) but would not actually change the pressed state of the button (i.e. `$Button.pressed` would not change).

If it does change the value of `$Button.pressed` but is not the same as setting `$Button.pressed` manually, then it's fairly ambiguous and confusing IMO.

If the only thing that this does is that it doesn't emit the `toggled` signal, then I would name it `set_pressed_no_signal`, and I would clarify the part about "Use only when you want to change the visual state of the button" which doesn't seem consistent with actually changing the value of `pressed` in the struct.

Reply:
Renamed to `set_pressed_no_signal`.